CHANGELOG.rst

```diff
@@ -55,6 +55,8 @@ Completions
   - brightnessctl (:issue:`8758`)
   - rclone (:issue:`8819`)
   - tuned-adm (:issue:`8760`)
+- Improved completions for:
+  - lxc
```

Comment:
thanks for remembering the changelog but we generally don't want to mention updates to existing completions

Reply:
I stopped adding them because it became overwhelming!